### ISSUE TITLE
Una historial por distrito/municipio/estado

### ISSUE THREAD

**Issue by hyeomans** [hyeomans/quien-me-representa-mx#6]
Una página que te muestre un histórico de:

* Municipio
* Distrito local
* Distrito federal
* Gobierno estatal

Dentro de esta página se mostraría el mapa, desplegando el polígono sobre el mapa de Openstreet map. Para sacar el centro usaríamos: [ST_Centroid.](https://postgis.net/docs/ST_Centroid.html)

**Comment by keogh** [comment]
A que te refieres con "historial" una lista de todos los que han estado en cargos públicos por municipio/distrito/etc?

**Comment by hyeomans** [comment]
Sí, pero no se si todos los cargos públicos por municipio. Esta complicado traer toda la información de todos los municipios.

Pero de distritos locales y federales si se puede. 

De los estados, es lo mismo. Cada estado tiene sus instituciones que varían de estado a estado.
